src/components/Common/UserAutocompleteFormField.tsx

```diff
@@ -80,6 +80,7 @@ export default function UserAutocomplete(props: UserSearchProps) {
         )}
         optionLabel={formatName}
         optionIcon={userOnlineDot}
+        optionImage={(option) => option.read_profile_picture_url || ""}
```

Comment:
![Screenshot from 2024-10-24 11-48-48](https://github.com/user-attachments/assets/a3b8bfaa-ba69-4e83-a5bf-66990be8df43)
Removing emtpy string would lead to an error